### ISSUE TITLE
Windows 10 fails to run "docker run hello-world" after upgrading to latest stable version

### ISSUE THREAD

**Issue by sudeshnasengupta** [docker/for-win#4431]
After upgrading to the latest stable release for Windows 10, I no longer am able to run the docker engine.

PS C:\> docker --version
Docker version 19.03.1, build 74b1e89
PS C:\> docker run hello-world
Unable to find image 'hello-world:latest' locally
C:\Program Files\Docker\Docker\Resources\bin\docker.exe: Error response from daemon: Get https://registry-1.docker.io/v2/: dial tcp: lookup registry-1.docker.io on [::1]:53: read udp [::1]:51849->[::1]:53: read: connection refused.
See 'C:\Program Files\Docker\Docker\Resources\bin\docker.exe run --help'.

  - [x] I have tried with the latest version of my channel (Stable or Edge)
  - [x] I have uploaded Diagnostics
  - Diagnostics ID: CAD24174-1DCE-4B93-9694-416B4DD1A23E/20190805233903

### Actual behavior
Error in accessing docker hub from shell, even though I can access via browser.  I've verified Windows Firewall inbound/outbound rules expose all ports in all profiles for both TCP and UDP.

Prior to upgrading, I was able to use docker for windows without any problems. Do I also need to update Hyper-V?

**Comment by mat007** [comment]
Sorry, for an unknown reason your diagnostics file never reached us.
Do you think you could try and send a fresh one?

**Comment by sudeshnasengupta** [comment]
Hi mat007: Since I needed to get docker for Windows working again, I just downgraded (from the latest 2.1.x) to: Docker for Desktop Version 2.0.0.0-win81 (29211).  When I get a chance again, I'll re-install and re-post here a new diagnostics file.  Thanks!

**Comment by BillzGitHub** [comment]
Hi all,

New to docker (and to posting on github).

I figured this was related to my issue so I would just post here rather than create a new ticket. 

I tried running the `docker run hello-world` command in both PowerShell and Command Prompt (ran both under Admin and non-Admin). I'm running Windows 10 Pro, Version 1903, OS Build 18362.239

Here is the error output I'm receiving:

`C:\Program Files\Docker\Docker\Resources\bin\docker.exe: Error response from daemon: hcsshim::CreateComputeSystem 9277294d11ffd68ae6aeac59602f7d3d33623a95a123e6c417376e91e768dc97: The virtual machine could not be started because a required feature is not installed.
(extra info: {"SystemType":"Container","Name":"9277294d11ffd68ae6aeac59602f7d3d33623a95a123e6c417376e91e768dc97","Owner":"docker","IgnoreFlushesDuringBoot":true,"LayerFolderPath":"C:\\ProgramData\\Docker\\windowsfilter\\9277294d11ffd68ae6aeac59602f7d3d33623a95a123e6c417376e91e768dc97","Layers":[{"ID":"cc6b0a2d-da7a-58ee-bfb6-f2e197079779","Path":"C:\\ProgramData\\Docker\\windowsfilter\\6a155457874d3faacbb71733f94a6fe8a496281075ae7f52ccb1120d2f1de588"},{"ID":"f975078a-4f04-50fa-8134-8a57868d8c4e","Path":"C:\\ProgramData\\Docker\\windowsfilter\\85b3bcd5ddd450c659a445ccc5fcb20e55766c068936cb2d73d7db00e96fc776"},{"ID":"3005d7e6-6371-5b6e-8226-afa3528ac367","Path":"C:\\ProgramData\\Docker\\windowsfilter\\79a1f35641b00e4350c9e7ec737302e8022fc56b2c9badc34127af995f9f6093"},{"ID":"4175340f-2a13-50d4-b5a1-e8e4a1a23908","Path":"C:\\ProgramData\\Docker\\windowsfilter\\235393f1653f5fad578d768d10d1180a160d91a5b693792cbea12461c7419f89"}],"HostName":"9277294d11ff","HvPartition":true,"EndpointList":["BBFA4511-DE0C-4F1F-9202-79DF536978A2"],"HvRuntime":{"ImagePath":"C:\\ProgramData\\Docker\\windowsfilter\\79a1f35641b00e4350c9e7ec737302e8022fc56b2c9badc34127af995f9f6093\\UtilityVM"},"AllowUnqualifiedDNSQuery":true}).`

I haven't found any reason for this error on stackoverflow, docker forums, or here.

Any help is greatly appreciated. Also let me know how or if I need to improve this post or submit a new ticket.

Thanks,
Bill
